lisa/tools/ls.py

```diff
@@ -106,7 +106,7 @@ def list(self, path: str, sudo: bool = False) -> List[str]:
             sudo=sudo,
         )
         if output:
-            return output.split()
+            return List(output.split())
```

Comment:
@SRIKKANTH  What is the reason for doing `List()` here ? It actually is not instantiate-able. It throws following error.

 ```
 File "XYZ/lisa/lisa/tools/ls.py", line 109, in list
    return List(output.split())
  File "XYZ/Python/3.10.15/x64/lib/python3.10/typing.py", line 955, in __call__
    raise TypeError(f"Type {self._name} cannot be instantiated; "
TypeError: Type List cannot be instantiated; use list() instead

```

Reply:
ok, I see this is fixed in main now.